.Rhistory

```diff
@@ -0,0 +1,62 @@
+hello()
```

Comment:
I guess you can delete your Rhistory from github since it has no relevance for the package